Sources/Model/TransferMethod/HyperwalletTransferMethod.swift

```diff
@@ -129,6 +129,9 @@ public class HyperwalletTransferMethod: Codable {
 
         // PayPal account related fields
         case email
+
+        // prepaid card related fields
+        case cardPackage
```

Comment:
need to add userToken for prepaidCard as well

Reply:
@jsaini-hw Why do we need to add userToken for prepaidCard? userToken is not a property of prepaidCard object, and I don't see it in the prepaidCard response

Reply:
It's in java-sdk and I checked HW2 code, it is required for instant issue cards. check with @whersbach-hw if we require it